Framework/Backend/log/InfoLoggerSender.js

```diff
@@ -15,19 +15,21 @@
 const {access, constants: {X_OK}} = require('fs');
 const {execFile} = require('child_process');
 const InfoLoggerMessage = require('./InfoLoggerMessage.js');
+const {LogLevel} = require("./LogLevel");
+const {LogSeverity} = require("./LogSeverity");
 
 /**
  * Sends logs as InfoLogger objects to InfoLoggerD over UNIX named socket
  * @docs https://github.com/AliceO2Group/InfoLogger/blob/master/doc/README.md
  */
 class InfoLoggerSender {
   /**
-   * @param {winston.instance} winston - local winston instance object
+   * @param {Logger} winston - local winston instance object
    */
-  constructor(winston, label = '') {
+  constructor(winston) {
     this._isConfigured = false;
     this.winston = winston;
-    this.label = label;
+    const label = 'infologger';
```

Comment:
I am not sure that this should default to 'infologger' without option to change it or provide your own label for winston.
With this change my logs will look like this:
```
2024-07-19T09:18:21.706Z [cog/server] info: Server listening on port 8080
2024-07-19T09:18:21.707Z [infologger] debug: InfoLogger executable not found
```
I think it should still be `[cog/infologgersender]` or some user preferred label.

Reply:
Do you instanciate `InfologgerSender` in other places than Logger?
Because if not, previous behavior was using `this.label` from static function which was `undefined` if I am not mistaken

Reply:
I do not think it was undefined. It was sending the label, but your change remove it: https://github.com/AliceO2Group/WebUi/pull/2494/files/5a03006411e5562b52e3e2665e1e6550330d054f#diff-98bf251bc3cf2b1739ad449dbca43568ef8be1aa4e2d3197bc3197af329c24f1L58